implementation/clabbert/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM openjdk:21-jdk AS build
```

Comment:
Huh, thats weird, alpine supports `openjdk-21` but there is no image with alpine and `openjdk-21`. By default it runs Oracle Linux.